src/lib/y2storage/storage_manager.rb

```diff
@@ -190,12 +190,13 @@ def staging_changed?
     # staging devicegraph.
     #
     # Beware: this method can cause data loss
-    def commit
+    def commit(force_rw: false)
       # Tell FsSnapshot whether Snapper should be configured later
       Yast2::FsSnapshot.configure_on_install = configure_snapper?
       callbacks = Callbacks::Commit.new
       storage.calculate_actiongraph
-      storage.commit(callbacks)
+      commit_options = ::Storage::CommitOptions.new(force_rw)
```

Comment:
Maybe `CommitOptions` should have its own wrapper class.

Reply:
As we do not need any special behavior (it is just some kind of 'struct' to pass options), I would keep it simple by now.